content/applications/hr/fleet/new_vehicle.rst

```diff
@@ -36,84 +49,248 @@ This section of the vehicle form relates to the person who is currently driving
+:guilabel:`Create Future Driver` form appears, depending on which field initiated the form.
+
+Both the :guilabel:`Create Driver` and :guilabel:`Create Future Driver` forms are identical.
 
-.. important::
-   A driver does **not** have to be an employee, but a driver must be listed in the *Contacts*
-   application. When creating a new driver, the driver is added to the *Contacts* application, not
-   the *Employees* application.
+.. _fleet/new_vehicle/general-info:
+
+.. note::
+   Depending on the installed applications, different tabs or fields may be visible on the
+   :guilabel:`Create Driver` and :guilabel:`Create Future Driver` forms.
+
+General information
+*******************
+
+Fill out the following information on the top-half of the form:
+
+- :guilabel:`Individual` or :guilabel:`Company`: choose if the driver being added is an individual
+  driver or a company. Click the radio button to make a selection.
+
+  When a selection is made, some fields may disappear from the form. If any of the fields below are
+  not visible, that is because :guilabel:`Company` was selected instead of :guilabel:`Individual`.
+- :guilabel:`Name`: enter the name of the driver or company in this field.
+- :guilabel:`Company Name`: using the drop-down menu, select the company the driver is associated
+  with.
+
+  If :guilabel:`Company` is selected for the :guilabel:`Individual` or :guilabel:`Company` field,
+  this field does not appear.
+- :guilabel:`Contact`: enter the contact information in this section.
+
+  If desired, the :guilabel:`Contact` field can be changed to a different type of contact. Click on
+  :guilabel:`Contact` to reveal a drop-down menu. The available options to select are
+  :guilabel:`Contact`, :guilabel:`Invoice Address`, :guilabel:`Delivery Address`,
+  :guilabel:`Follow-up Address`, or :guilabel:`Other Address`.
+
+  If desired, select one of these other options for the :guilabel:`Contact` field, and enter the
+  corresponding information.
+
+  If :guilabel:`Company` is selected for the  :guilabel:`Individual` or :guilabel:`Company` field,
+  this field is labeled :guilabel:`Address` and cannot be modified.
+
+- :guilabel:`Tax ID`: enter the driver or company's tax ID in this field.
+- :guilabel:`Job Position`: enter the driver's job position in this field. If :guilabel:`Company` is
+  selected for the :guilabel:`Individual` or :guilabel:`Company` field, this field does not appear.
+- :guilabel:`Phone`: enter the driver or company's phone number in this field.
+- :guilabel:`Mobile`: enter the driver or company's mobile number in this field.
+- :guilabel:`Email`: enter the driver or company's email address in this field.
+- :guilabel:`Website`: enter the driver or company's website address in this field.
+- :guilabel:`Title`: using the drop-down menu, select the driver's title in this field. The default
+  options are :guilabel:`Doctor`, :guilabel:`Madam`, :guilabel:`Miss`, :guilabel:`Mister`, and
+  :guilabel:`Professor`.
+
```

Comment:
I'm also seeing a Language field too -- does that not appear for you?

Reply:
That's from the Employees app (I believe).